src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.tsx

```diff
@@ -254,6 +255,23 @@ export const ConversationTabs = ({
             />
           </>
         )}
+
+        <div className="conversations-sidebar-divider" />
+
+        <div className="conversations-sidebar-title" css={{marginBlock: '32px 0'}}>
+          {t('meetings.navigation.parent.label')}
+        </div>
+
+        <ConversationTab
```

Comment:
Wrap the buttons in a container with role="tablist" and aria-label

` <div role="tablist" aria-label={t('meetings.navigation.parent.label)}>`

Reply:
ahh didn't know that, no need to add then, aria-label is still useful though. you can play voice over and test if its working fine